lua/apisix/plugin.lua

```diff
@@ -191,7 +191,7 @@ function fetch_api_routes()
                            core.json.delay_encode(api_routes, true))
             for _, route in ipairs(api_routes) do
                 core.table.insert(routes, {
-                        method = route.methods,
+                        methods = route.methods,
```

Comment:
I have no idea which one is correct, @membphis Please have a check :)

Reply:
the detail issue [issues 750](https://github.com/iresty/apisix/issues/750)